templates/Pages/rules_submission.php

```diff
@@ -65,6 +65,14 @@
             data, it must ensure that data is persistently stored before
             acknowledging the close.
         </li>
+        <li>
+            Data and metadata must be written in its entirety and not reduced based
+            on its contents. The goal of the io500
+            is to provide dataset independent performance results, and techniques
+            such as deduplication, compression, etc would bias performance as the
+            io500 benchmark uses partially predictable content that is not realistic
```

Comment:
I would replace realistic with: "representative of existing workloads".

Reply:
I would write "dataset-independent", and "etc.", but not a huge deal.  I think someone else already pointed out "IO500".